src/nft/components/bag/BagTokenSelectorModal.tsx

```diff
@@ -0,0 +1,140 @@
+import { Trans } from '@lingui/macro'
```

Comment:
the bag folder is getting fairly large, can we add a folder for the token selector components?